File: TheaterSuggestions/CSharp/SeatsSuggestions.Domain/NotSuggestionMatchedExpectation.cs
using System.Collections.Generic;

namespace SeatsSuggestions.Domain
{
    internal class NotSuggestionMatchedExpectation : SuggestionMade
    {
        public NotSuggestionMatchedExpectation(int partyRequested, PricingCategory pricingCategory) : base(
            partyRequested, pricingCategory, new List<Seat>())
        {
        }
    }
}

File: TheaterSuggestions/Java/SeatsSuggestionsDomain/src/main/java/com/baasie/SeatsSuggestionsDomain/NotSuggestionMatchedExpectation.java
package com.baasie.SeatsSuggestionsDomain;

import java.util.ArrayList;

public class NotSuggestionMatchedExpectation extends SuggestionMade {
    public NotSuggestionMatchedExpectation(PartyRequested partyRequested, PricingCategory pricingCategory) {
        super(new ArrayList<>(), partyRequested, pricingCategory);
    }
}
